ui_components/setup_wizard.py

```
@@ -134,15 +144,47 @@ def _step_database_connection(self):
+                    
+                    if final_success:
+                        # Clear any cached schema and semantic layer information
+                        keys_to_clear = ['schema_analysis_done', 'schema_info', 'semantic_layer_built', 'semantic_method', 'generated_metadata', 'sample_queries']
+                        for key in keys_to_clear:
+                            if key in st.session_state:
+                                del st.session_state[key]
+                        
+                        # Clear semantic layer vector store
+                        try:
+                            enhanced_semantic_layer.clear_collections("default")
+                        except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)